src/iframeResizer.contentWindow.js

```diff
@@ -292,7 +292,7 @@
 
 			window.parentIFrame = {
 				close: function closeF(){
-					sendSize('close','parentIFrame.close()', 0, 0);
+					sendSize('close','parentIFrame.close()', Infinity, Infinity);
```

Comment:
@davidjbradshaw the problem isn't with the host page, the problem is that the host page never even receives the event because of the function I referenced here on line [577](https://github.com/davidjbradshaw/iframe-resizer/blob/master/src/iframeResizer.contentWindow.js#L577) `isSizeChangeDetected()`.

```
function isSizeChangeDetected(){
	function checkTolarance(a,b){
		var retVal = Math.abs(a-b) <= tolerance;
		return !retVal;
	}

	currentHeight = (undefined !== customHeight)  ? customHeight : getHeight[heightCalcMode]();
	currentWidth  = (undefined !== customWidth )  ? customWidth  : getWidth();

	return	checkTolarance(height,currentHeight) ||
			(calculateWidth && checkTolarance(width,currentWidth));
}
```

Which is defined as above. When `currentHeight` and `customHeight` are both 0 `checkTolarance` and thus `isSizeChangeDetected` both return `false`. If no change is detected then no event is propagated/sent to the host page.  My fix sets `customHeight` to `Infinity` which essentially guarantees `isSizeChangeDetected` will always return `true` when a close event is triggered.